sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -569,3 +570,29 @@ public abstract partial class ObjectSerializer
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
 }
+namespace Azure.Messaging
+{
+    [System.Text.Json.Serialization.JsonConverterAttribute(typeof(Azure.Messaging.CloudEventConverter))]
+    public partial class CloudEvent
+    {
+        public CloudEvent(string source, string type, object jsonSerializable, System.Type dataSerializationType = null) { }
+        public CloudEvent(string source, string type, System.ReadOnlyMemory<byte> data, string dataContentType) { }
+        public string DataContentType { get { throw null; } set { } }
+        public string DataSchema { get { throw null; } set { } }
+        public System.BinaryData EventData { get { throw null; } }
+        public System.Collections.Generic.IDictionary<string, object> ExtensionAttributes { get { throw null; } }
+        public string Id { get { throw null; } set { } }
+        public string Source { get { throw null; } set { } }
```

Comment:
There is still a chance we make Source and DataSchema Uris. The issue is that this will be a problem when deserializing values that don't match the spec, hence why string might be the best we can do.